packages/app-desktop/package.json

```diff
@@ -95,7 +95,7 @@
       "icon": "../../Assets/LinuxIcons",
       "category": "Office",
       "desktop": {
-        "Icon": "joplin",
+        "Icon": "@joplinapp-desktop",
```

Comment:
This doesn't help atm - still waiting to hear from the owner of the AppImageLauncher but figured I'd do it for good order